docs/user-guide/global-configurations/cluster-and-environments.md

```diff
@@ -42,17 +42,33 @@ Provide the endpoint/URL of your kubernetes cluster.It is recommended to use a s
 
 **\(b\) Easy cluster migrations -** Cluster url is given in the name of the cloud provider used, so migrating your cluster from one provider to another will result in waste of time and effort. On the other hand, if using a self-hosted url migrations will be easy as the url is of single hosted domain independent of the cloud provider.
 
+To get the **`Server URL`**, run the following the command :
+
+```bash
+curl -O https://raw.githubusercontent.com/devtron-labs/utilities/main/kubeconfig-exporter/kubernetes_export_sa.sh && bash kubernetes_export_sa.sh cd-user devtroncd https://raw.githubusercontent.com/devtron-labs/utilities/main/kubeconfig-exporter/clusterrole.yaml
```

Comment:
That's not the command to get the server URL

Reply:
Corrected